spf.go

```diff
@@ -113,6 +113,17 @@ type Resolver interface {
 	LookupPTR(string) ([]string, *ResponseExtras, error)
 }
 
+func (x *ResponseExtras) GetTTL() time.Duration {
+	if x == nil {
+		return 0
+	}
+	return x.TTL
+}
+
+func (x *ResponseExtras) IsVoid() bool {
+	return x != nil && x.Void
+}
+
```

Comment:
Nope, having two different access models worsen the design.
We either have to use public fields (`TTL` and `Void`) and no getters or private fields (`ttl` and `void`) and getters (`TTL()` and `Void()`).

In the latter, we also need the constructor.

Also, it is better to place methods next to the structure

Reply:
done